.github/workflows/on_pr_master.yml

```diff
@@ -20,6 +20,8 @@ jobs:
     steps:
       # For pull requests it's not necessary to checkout the code
       - uses: actions/checkout@v2
+        with:
+          submodules: recursive
```

Comment:
I guess this isn't required as i understand we won't be making any changes to files under /template from vagrant-hashistack repo so we basically do not need this check.